src/samples/test-bots/TeamsSSOAgent/README.md

```diff
@@ -0,0 +1,52 @@
+﻿# Teams Conversation Bot
+
+Teams AI  Conversation Bot sample for Teams.
```

Comment:
bot -> agent

Reply:
This is POC, Rido... not the final deal.  It is for comment on what it's doing.